tools/bundle-build/README.md

```diff
@@ -0,0 +1,15 @@
+## Bundle Build
+
+Builds the OpenSearch bundle.
+
+The OpenSearch repo is built first, followed by common-utils. These dependencies are published to maven local, and subsequent project builds pick up those.
```

Comment:
Seems like this is based on the order in the manifest rather than a special case, is that correct?

Reply:
Correct, it will build in the order defined in the manifest.  I called out these two specifically because almost all components rely on these being built and available in order to compile.